packages/core/src/blocks/File/helpers/render/createAddFileButton.ts

```diff
@@ -36,11 +37,7 @@ export const createAddFileButton = (
   };
   // Opens the file toolbar.
   const addFileButtonClickHandler = () => {
-    editor.transact((tr) =>
-      tr.setMeta(editor.filePanel!.plugins[0], {
-        block: block,
-      }),
-    );
+    editor.getExtension(FilePanelPlugin)?.showMenu(block.id);
```

Comment:
minor; I think better to throw an error (! instead of ?) if the extension is not there